tests/ftests/051-sudo-set_uid_gid_v2.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# Test to set the uid/gid on a cgroup using the python bindings
```

Comment:
Do you think adding `Cgroup v2` to the test description, might help. Should the copyright year also need to change?

Reply:
Good idea on both accounts.